readthedocs/projects/validators.py

```diff
@@ -12,39 +10,13 @@
 from django.utils.translation import ugettext_lazy as _
 
 
-domain_regex = (
-    r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}(?<!-)\.?)|'
-    r'localhost|'  # localhost...
-    r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|'  # ...or ipv4
-    r'\[?[A-F0-9]*:[A-F0-9:]+\]?)'  # ...or ipv6
-)
-
-
 @deconstructible
 class DomainNameValidator(RegexValidator):
     message = _('Enter a valid plain or internationalized domain name value')
-    regex = re.compile(domain_regex, re.IGNORECASE)
-
-    def __init__(self, accept_idna=True, **kwargs):
-        message = kwargs.get('message')
-        self.accept_idna = accept_idna
-        super().__init__(**kwargs)
-        if not self.accept_idna and message is None:
-            self.message = _('Enter a valid domain name value')
-
-    def __call__(self, value):
-        try:
-            super().__call__(value)
-        except ValidationError as exc:
-            if not self.accept_idna:
-                raise
-            if not value:
-                raise
-            try:
-                idnavalue = value.encode('idna')
-            except UnicodeError:
-                raise exc
-            super().__call__(idnavalue)
```

Comment:
What's up with this removal? I don't really even understand this code, was the `accept_idna` attribute used at all?

Reply:
Nope, this was just copy/pasta from the original PR in django itself, we aren't using it.

Reply:
I just double checked corporate and ext, we are good

Reply:
Forgot to ping you on this one @stsewd :point_up:

Reply:
Already replied if that's what you mean p: maybe github didn't update for you?

Reply:
oops! yeah, github didn't update for me